src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,503 @@
+                        "shortDescription": "                   a"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """{
+                    "validationErrors": {
+                        "$.codeValue": [
+                            "codeValue cannot contain leading or trailing spaces."
+                        ],
+                        "$.namespace": [
+                            "namespace cannot contain leading or trailing spaces."
+                            ],
+                        "$.shortDescription": [
+                            "shortDescription cannot contain leading or trailing spaces."
+                        ]
+                    },
+                    "errors": [],
+                    "detail": "Data validation failed. See 'validationErrors' for details.",
+                    "type": "urn:ed-fi:api:bad-request:data",
+                    "title": "Data Validation Failed",
+                    "status": 400,
+                    "correlationId": null
+                  }
+                  """
+
+        # Descriptors are not validating the whitespace yet. DMS-295
+        @ignore
+        Scenario: 08 Ensure clients cannot create a descriptor with trailing spaces in required attributes
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "a                      ",
+                        "description": "a                    ",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "a                   "
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """{
+                    "validationErrors": {
+                        "$.codeValue": [
+                            "codeValue cannot contain leading or trailing spaces."
+                        ],
+                        "$.namespace": [
+                            "namespace cannot contain leading or trailing spaces."
+                            ],
+                        "$.shortDescription": [
+                            "shortDescription cannot contain leading or trailing spaces."
+                        ]
+                    },
+                    "errors": [],
+                    "detail": "Data validation failed. See 'validationErrors' for details.",
+                    "type": "urn:ed-fi:api:bad-request:data",
+                    "title": "Data Validation Failed",
+                    "status": 400,
+                    "correlationId": null
+                  }
+                  """
+
+        @ignore
+        Scenario: 09 Ensure clients can create a descriptor using optional attributes
```

Comment:
This scenario is essentially a repeat of scenario 1. Perhaps this one could be removed.

Reply:
Removed